lib/_components/_all.scss

```diff
@@ -1,4 +1,5 @@
 @import "appArticle/article";
 @import "appCard/card";
+@import "betaBanner/beta-banner";
```

Comment:
Shouldn't we use the phase banner approach from gov design system?

https://design-system.service.gov.uk/components/phase-banner/